components/footer.tsx

```diff
@@ -2,13 +2,16 @@ import React, { ReactNode } from "react";
 import {
   GitHubLogoIcon,
   LinkedInLogoIcon,
-  TwitterLogoIcon,
 } from "@radix-ui/react-icons";
 import Link from "next/link";
 
 export default function Footer() {
   const currentYear = new Date().getFullYear();
-
+  const TwitterX = () => (
+    <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px" width="25" height="25" viewBox="0,0,256,256">
+      <g transform=""><g fill="none" fill-rule="nonzero" stroke="none" stroke-width="none" stroke-linecap="butt" stroke-linejoin="none" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" ><path transform="scale(8.53333,8.53333)" d="M24,4c1.105,0 2,0.895 2,2v18c0,1.105 -0.895,2 -2,2h-18c-1.105,0 -2,-0.895 -2,-2v-18c0,-1.105 0.895,-2 2,-2z" id="strokeMainSVG" fill="#190d0d" stroke="#190d0d" stroke-width="3" stroke-linejoin="round"></path><g transform="scale(8.53333,8.53333)" fill="#ffffff" stroke="none" stroke-width="1" stroke-linejoin="miter"><path d="M6,4c-1.105,0 -2,0.895 -2,2v18c0,1.105 0.895,2 2,2h18c1.105,0 2,-0.895 2,-2v-18c0,-1.105 -0.895,-2 -2,-2zM8.64844,9h4.61133l2.69141,3.84766l3.33008,-3.84766h1.45117l-4.12891,4.78125l5.05078,7.21875h-4.61133l-2.98633,-4.26953l-3.6875,4.26953h-1.47461l4.50586,-5.20508zM10.87891,10.18359l6.75391,9.62695h1.78906l-6.75586,-9.62695z"></path></g></g></g>
```

Comment:
Dont place the SVG here. Make a new file in `public/icons/TwitterIcon.svg` , place it there and import

Reply:
Well it does not support as ReactNode in other file as it  is void, it will fail

Reply:
I did what u said but it gives error for reactNode not support for void

Reply:
Can you search for icons like we did with radix icons? This doesnt look good in code

Reply:
Radix does not have new twitter icon , I search their firstly

Reply:
https://fontawesome.com/icons/x-twitter?f=brands&s=solid

Reply:
![image](https://github.com/ashutosh-rath02/git-re/assets/89975347/f0f914f5-419b-419f-b024-2a08775799e3)

Looks like it is not working

Reply:
You need to debug it dude or find alternatives.😬